Not supported with three@"^0.153.0"

Could not resolve dependency:
npm ERR! peer three@"^0.144.0" from three-bvh-csg@0.0.7
npm ERR! node_modules/three-bvh-csg
npm ERR!   three-bvh-csg@"*" from the root project

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
   },
   "homepage": "https://github.com/gkjohnson/three-bvh-csg#readme",
   "peerDependencies": {
-    "three": "^0.144.0",
+    "three": ">=0.144.0",
     "three-mesh-bvh": "^0.6.0"
   },
   "devDependencies": {

